### DISCUSSION TITLE
ArraySchema의 사용법이 궁금해요.

### DISCUSSION THREAD

**Discussion by kyn320** [naverz/zepeto-studio-kor#643] [5. Q&A]
DataStorage 를 이용해서 Array 타입의 스키마를 저장하려고 하는데요.
기존에 나온 샘플들에도 Array 처리 예제가 따로 없더라구요.
혹시 사용법이 나온 가이드가 있거나, 없다면 사용법을 알 수 있을까요?

![image](https://user-images.githubusercontent.com/11557205/164153366-1b7f755d-20ec-45a9-bb13-e819bae3e3fb.png)

![image](https://user-images.githubusercontent.com/11557205/164153485-446d5351-b60b-4610-b936-584bbff5bfa6.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 문의주신 구조는 JSON을 이용해서 구현하실 수 있습니다.
간단한 예제 코드 안내 드립니다.

<img width="621" alt="스크린샷 2022-04-22 오후 5 08 13" src="https://user-images.githubusercontent.com/100118650/164648382-b97caa4f-9887-4633-ac10-b9f636bb364b.png">
